wear/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,4 @@
 <resources>
-  <string name="hello_world">Hello Wear World!</string>
+  <string name="empty_events">No contacts events found</string>
+  <string name="complications_provider_contact_events">Contact Events</string>
```

Comment:
This would have to be extracted into it's own `contactevents_complication-resource.xml` file which would like to a `@string/Contact_Events` in `strings.xml`

Same goes to the strings above